generate-release/src/helpers.rs

```diff
@@ -19,21 +19,16 @@ pub fn get_merged_prs(
         .context("Failed to get commits")?;
     println!("Found {} commits", commits.len());
 
-    println!("Getting list of all merged PRs from {from} to {to} with label {label:?}");
+    println!("Getting list of all merged PRs with label {label:?}");
 
     let base_commit = client.get_commit(from, BevyRepo::Bevy)?;
     let base_commit_date = &base_commit.commit.committer.date[0..10];
 
     // We also get the list of merged PRs in batches instead of getting them separately for each commit
-    let prs = client.get_issues_and_prs(
-        BevyRepo::Bevy,
-        IssueState::Merged,
-        Some(base_commit_date),
```

Comment:
It's not totally clear to me why this helps, based on GH's [documentation](https://docs.github.com/en/rest/issues/issues?apiVersion=2022-11-28#list-repository-issues) about the `since` param in their API, but it does seem to cause more stuff to get generated.